src/boot/efi/log.c

```diff
@@ -60,6 +74,19 @@ EFI_STATUS log_internal(LogLevel level, EFI_STATUS status, const char *format, .
                 ST->ConOut->OutputString(ST->ConOut, (char16_t *) u"\r\n");
         ST->ConOut->SetAttribute(ST->ConOut, log_colors[level]);
 
+        EFI_TIME time = {};
+        (void) RT->GetTime(&time, NULL);
+        printf_status(status,
+                      "[sd-boot %4u-%02u-%02u %02u:%02u:%02u %s:%u] ",
```

Comment:
I'd prefer if these messages would look similar to what we'd generate in userspace if you do SYSTEMD_LOG_LOCATION=1 SYSTEMD_LOG_TIME=1. i.e. the `[]` should go I think, and the sd-boot thing probably too, it's kinda redundant if you show the location